buildSrc/src/main/kotlin/kotlin-library-conventions.gradle.kts

```diff
@@ -8,8 +8,8 @@ plugins {
 tasks.withType<KotlinCompile>().configureEach {
 	kotlinOptions {
 		jvmTarget = Versions.jvmTarget.toString()
-		apiVersion = "1.1"
-		languageVersion = "1.1"
+		apiVersion = "1.3"
+		languageVersion = "1.3"
```

Comment:
Is this a breaking change, i.e. will users who use Kotlin 1.1 api/language level with JUnit 5.4.x still be able to use 5.5 with these changes?

Reply:
See https://kotlinlang.org/docs/reference/evolution/compatibility-modes.html

`languageVersion` only affects your implementation, the Kotlin runtime should provide the compatibility. Keeping it to 1.1 would enforce it.

`apiVersion` should also be fine as long as you don't use types that are only available in >1.1. Keeping it to `1.1` would enforce that.

So, yes, depending on how much you rely on new Kotlin features from 1.3 it might be a breaking change.